docs/organization/README.md

```diff
@@ -30,37 +30,27 @@ Follows a list and description of the repositories under .NET **nanoFramework**
-- [System.Device.I2c](https://github.com/nanoframework/System.Device.I2c) - this repo contains the Windows.Devices.I2c library that is used in developing C# applications, this replaces Windows.Devices.I2c.
-- [Windows.Devices.I2c](https://github.com/nanoframework/Windows.Devices.I2c) **:warning: Deprecated!** - this repo contains the Windows.Devices.I2c library that is used in developing C# applications.
-- [System.Device.Pwm](https://github.com/nanoframework/System.Device.Pwm) - this repo contains the System.Device.Pwm library that is used in developing C# applications.
-- [Windows.Devices.Pwm](https://github.com/nanoframework/Windows.Devices.Pwm) **:warning: Deprecated!** - this repo contains the Windows.Devices.Pwm library that is used in developing C# applications.
-- [System.IO.Ports](https://github.com/nanoframework/System.IO.Ports) - this repo contains the System.IO.Ports library that is used in developing C# applications.
-- [Windows.Devices.SerialCommunication](https://github.com/nanoframework/Windows.Devices.SerialCommunication) **:warning: Deprecated!** - this repo contains the Windows.Devices.SerialCommunication library that is used in developing C# applications.
-- [System.Device.Spi](https://github.com/nanoframework/System.Device.Spi)- this repo contains the System.Device.Spi library that is used in developing C# applications.
-- [Windows.Devices.Spi](https://github.com/nanoframework/Windows.Devices.Spi) **:warning: Deprecated!** - this repo contains the Windows.Devices.Spi library that is used in developing C# applications.
-- [Windows.Devices.WiFi](https://github.com/nanoframework/Windows.Devices.WiFi) - this repo contains the Windows.Devices.WiFi library that is used in developing C# applications.
-- [Windows.Storage](https://github.com/nanoframework/Windows.Storage) - this repo contains the Windows.Storage library that is used in developing C# applications.
-- [Windows.Storage.Streams](https://github.com/nanoframework/Windows.Storage.Streams) - this repo contains the Windows.Storage.Streams library that is used in developing C# applications.
-- [nanoFramework.Devices.Can](https://github.com/nanoframework/nanoFramework.Devices.Can) - this repo contains nanoFramework.Devices.Can Class Library.
-- [nanoFramework.Devices.OneWire](https://github.com/nanoframework/nanoFramework.Devices.OneWire) - this repo contains nanoFramework 1-Wire Class Library
+### System.Device Libraries
+- [System.Device.Dac](https://github.com/nanoframework/System.Device.Dac) - this repo contains the nanoFramework.System.Devices.Dac library that is used in developing C# applications that require DAC (Digital Analog Converter) capabilities.
+- [System.Device.Adc](https://github.com/nanoframework/System.Device.Adc) - this repo contains the `System.Device.Adc` library that is used in developing C# applications, **this replaces the deprecated `Windows.Devices.Adc` library**.
```

Comment:
Do we need to keep referring to the deprecated Win.Devices libraries? That's history. Don't think this bring much value here...